sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/SchemaRegistryAvroEncoder.cs

```diff
@@ -45,10 +46,10 @@ public SchemaRegistryAvroEncoder(SchemaRegistryClient client, string groupName,
         }
 
         // TODO support backcompat for first beta
```

Comment:
Is this comment still needed?

Reply:
Nope, I'll remove it once we remove the backcompat support in the next release.